src/main/java/com/segment/analytics/android/integrations/amplitude/AmplitudeIntegration.java

```diff
@@ -92,6 +92,11 @@ public AmplitudeClient get() {
     boolean trackSessionEvents = settings.getBoolean("trackSessionEvents", false);
     amplitude.trackSessionEvents(trackSessionEvents);
     logger.verbose("AmplitudeClient.getInstance().trackSessionEvents(%s);", trackSessionEvents);
+
+    boolean enableLocationListening = settings.getBoolean("enableLocationListening", true);
```

Comment:
Does this default to `true`?

Reply:
We should set the default to whatever is in metadata.

Reply:
For context, Amplitude defaults this setting to true, so I think we probably should as well to emulate native behavior.

Reply:
Makes sense. The default in metadata should be the default behaviour of their SDK, and should be the same value here as well.